configure.ac

```diff
@@ -3735,6 +3735,11 @@ AS_VAR_IF([with_system_expat], [yes], [
 AC_SUBST([LIBEXPAT_CFLAGS])
 AC_SUBST([LIBEXPAT_INTERNAL])
 
+AS_VAR_IF(
+  [ac_cv_header_sys_time_h], [yes],
+  [AS_VAR_SET([have_libexpat], [yes])]
+)
```

Comment:
I'm a little lost here.  We're setting `have_libexpat=yes` if we have `sys/time.h`?

Reply:
I see now that `have_libexpat` is a poorly chosen name. The initial version simply used `ac_cv_header_sys_time_h` for the `PY_STDLIB_MOD` checks. The rationale was that if the deps changed, I could simply fix `have_libexpat` instead of patching the two `PY_STDLIB_MOD` checks.

Reply:
Rewrote to something more explicit in 92350d7846d780a8b2ce35c6909fc183ae76c520